erpnext_com/www/docs/user/manual/en/loan-management/loan-application.md

```diff
@@ -0,0 +1,50 @@
+<!-- add-breadcrumbs -->
+# Loan Application
+> Introduced in Version 13
+
+**A Loan Application contains details about the applicant and loan security details for review.**
+
+During a loan process, the first step a customer or employee has to do is submit a loan application for review. In case of a secured loan, a loan application can also contain proposed loan securities.
```

Comment:
```suggestion
During a loan process, the first step a customer or employee has to do is submit a Loan Application for review. In case of a secured loan, a Loan Application can also contain proposed loan securities.
```